README.md

```diff
@@ -1,1100 +1,25 @@
+[**Quickstart**](https://learn.microsoft.com/azure/azure-app-configuration/quickstart-feature-flag-dotnet): A quickstart guide is available to learn how to integrate feature flags from *Azure App Configuration* into your .NET applications.
 
-Here are some of the benefits of using this library:
+[**Feature Reference**](https://learn.microsoft.com/azure/azure-app-configuration/feature-management-dotnet-reference?pivots=preview-version): This document provides a full feature rundown.
 
-* A common convention for feature management
-* Low barrier-to-entry
-  * Built on `IConfiguration`
-  * Supports JSON file feature flag setup
-* Feature Flag lifetime management
-  * Configuration values can change in real-time, feature flags can be consistent across the entire request
-* Simple to Complex Scenarios Covered
-  * Toggle on/off features through declarative configuration file
-  * Dynamically evaluate state of feature based on call to server
-* API extensions for ASP.NET Core and MVC framework
-  * Routing
-  * Filters
-  * Action Attributes
+[**API reference**](https://go.microsoft.com/fwlink/?linkid=2091700): This API reference details the API surface of the libraries contained within this repository.
 
-**API Reference**: https://go.microsoft.com/fwlink/?linkid=2091700
+## Examples
 
-## Index
-* [Feature Flags](#feature-flags)
-    * [Feature Filters](#feature-filters)
-    * [Feature Flag Declaration](#feature-flag-declaration)
-* [Consumption](#consumption)
-* [ASP.NET Core Integration](#ASPNET-core-integration)
-* [Implement a Feature Filter](#implementing-a-feature-filter)
-* [Providing a Context For Feature Evaluation](#providing-a-context-for-feature-evaluation)
-* [Built-in Feature Filters](#built-in-feature-filters)
-* [Targeting](#targeting)
-  * [Targeting Exclusion](#targeting-exclusion)
-* [Variants](#variants)
-    * [Variants in Dependency Injection](#variants-in-dependency-injection)
-* [Telemetry](#telemetry)
-    * [Enabling Telemetry](#enabling-telemetry)
-    * [Custom Telemetry Publishers](#custom-telemetry-publishers)
-    * [Application Insights Telemetry Publisher](#application-insights-telemetry-publisher)
-* [Caching](#caching)
-* [Custom Feature Providers](#custom-feature-providers)
+* [.NET Console App](./examples/ConsoleApp)
+* [.NET Console App with Targeting](./examples/TargetingConsoleApp)
+* [ASP.NET Core Web App (Razor Page)](./examples/RazorPages)
+* [ASP.NET Core Web App (MVC)](./examples/FeatureFlagDemo)
+* [Blazor Server App](./examples/BlazorServerApp)
+* [Evaluation Data to Application Insights](./examples/EvaluationDataToApplicationInsights)

```

Comment:
@zhenlan @jimmyca15 @rossgrambo Any suggestion for the name of this example project?
I think this example shows the following thing:
1. How to use variant
2. How to use telemetry and collect it to App Insights
3. The general idea of how to do experimentation with FM lib and App Insights

Reply:
Yeah good call. We don't have full experimentation so I don't think we can mention that... 

Maybe just `VariantsWithTelemetry`?

Reply:
To me, it's an "ASP.NET Core Web App with Telemetry". People have to use variant feature flags to enable telemetry, so that's just implementation detail. Experimentation is a methodology to analyze telemetry. From the application perspective, it has no idea about the experimentation. So if you want to follow the naming of other examples, maybe "RazorPagesWithTelemetry"?

Reply:
I like Zhenlan's idea. I decide to land on "ASP.NET Core Web App with Feature Flag Telemetry"